2022q1/pot.adoc

```diff
@@ -0,0 +1,23 @@
+=== Containers & FreeBSD: Pot, Potluck & Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on github] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Stephan Lichtenauer (Potluck) <sl@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+As a result of production testing in a real-world cluster deployment, pot and related projects received stability improvements for controlling the pot lifecycle (i.e., pot prepare/start/stop). +
+Various attributes and commands have been developed to improve support of nomad orchestration and batch jobs (e.g., change dns config during clone, ability to disable tmpfs, new last-run-stats command). A new pot release will follow soon.
+
+Potluck aims to be to FreeBSD and pot what Dockerhub is to Linux and Docker: a repository of pot flavours and complete container images for usage with pot and in many cases nomad.
+
+Many of the core images like link:https://potluck.honeyguide.net/blog/nomad-server/[Nomad], link:https://potluck.honeyguide.net/blog/consul/[Consul] and link:https://potluck.honeyguide.net/blog/vault/[Vault] that can be used to build a private cloud and orchestration platform, but also e.g. link:https://potluck.honeyguide.net/blog/prometheus/[Prometheus] or link:https://potluck.honeyguide.net/blog/postgresql-patroni/[PostgreSQL Patroni], have reached a stable status over the last quarter and are in production use at a fintech now..
```

Comment:
"fintech"?

Reply:
Hi @pauamma, I agree, that is not relevant. I changed the section to "are in production use now".

Ok?